pkg/cluster/cache.go

```diff
@@ -103,6 +103,24 @@ func GetCachedToolchainCluster(name string) (*CachedToolchainCluster, bool) {
 	return clusterCache.getCachedToolchainCluster(name, true)
 }
 
+// GetClustersFunc a func that returns all the clusters from the cache
+type GetClustersFunc func(conditions ...Condition) []*CachedToolchainCluster
+
+// Clusters the func to retrieve all the clusters
+var Clusters GetClustersFunc = GetClusters
+
+// GetClusters returns the kube clients for all the clusters from the cache of the clusters
+func GetClusters(conditions ...Condition) []*CachedToolchainCluster {
```

Comment:
one more thing, `GetClusters` is pretty generic and can be confusing. How about:
* GetToolchainClusters
* GetCachedToolchainClusters

so it's clear which clusters we are talking about

Reply:
There is already a function named as `GetCachedToolchainClusters` , but to avoid confusion and for better understanding updating it to `GetToolchainClustersCachedFunc`